### PR TITLE
Improvements of the Readme

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**

### CODE REVIEW

File: dockerfiles/apim/README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ in order to obtain latest bug fixes and updates for the product.
 ##### 3. Build the Docker image.
 - Navigate to `<DOCKERFILE_HOME>` directory. <br>
   Execute `docker build` command as shown below.
-    + `docker build -t wso2am:2.1.0 .`
+    + `sudo docker build -t wso2am:2.1.0 .`
```
Comment (pubudu538) [inline dockerfiles/apim/README.md]: Let's not add sudo here in commands. Users has to configure to run docker in non root user.
https://docs.docker.com/engine/installation/linux/linux-postinstall/

Comment (ThilinaManamgoda) [reply dockerfiles/apim/README.md]: I will remove this

File: dockerfiles/apim/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ git clone https://github.com/wso2/docker-apim.git
 >The local copy of the `dockerfile` directory will be referred to as `DOCKERFILE_HOME` from this point onwards.
 
 ##### 2. Add JDK and WSO2 API Manager distributions to `<DOCKERFILE_HOME>/files`
-- Download [JDK 1.8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) 
+- Download [JDK 1.8(jdk-8u*-linux-x64.tar.gz)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
```
Comment (DilanUA) [inline dockerfiles/apim/README.md]: Please add proper spacing in between JDK 1.8 and (jdk-8u*-linux-x64.tar.gz).

Comment (ThilinaManamgoda) [reply dockerfiles/apim/README.md]: I have fixed this

**Pull request merged**